lambda/translcore.ml

```diff
@@ -1150,7 +1150,7 @@ and transl_letop ~scopes loc env let_ ands param case partial =
           transl_ident (of_location ~scopes and_.bop_op_name.loc) env
             and_.bop_op_type and_.bop_op_path and_.bop_op_val
         in
-        let exp = transl_exp ~scopes and_.bop_exp in
```

Comment:
I was originally unconvinced by the `_exp => _expr` rename (it adds noise to the diff for no obvious reason), but I looked at typedtree and it is true that `_expr` is used everywhere else, so this is an improvement. (It's usefeul to include such justifications in the PR description.)

Reply:
Sorry about the noise in the diff, should have comitted and explained it separately.

Reply:
No worries! I posted the explanation to help other people looking at the code.